### PR TITLE
Fix CLI --help for OS X

### PR DESCRIPTION
**Pull request by davidxia**

### CODE REVIEW

**Comment by davidxia** [comment]
I used [`sys.platform`][1] since `os.name` is `posix` for both linux and OS X. I didn't touch the `nt` part because I don't know the choice behind that. For Windows, the `sys.platform` doc says it can return the below:

```
Windows	'win32'
Windows/Cygwin	'cygwin'
```

  [1]: https://docs.python.org/2/library/sys.html#sys.platform

File: letsencrypt/error_handler.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 """Registers functions to be called if an exception or signal occurs."""
 import logging
+import sys
```
Comment (kuba) [inline letsencrypt/error_handler.py]: nit: we order imports alphabetically (and group them by stdlib/3rd-party/local sections)

**Comment by kuba** [comment]
LGTM. Thanks!

Could the merger fix above comment in case it's not resolved earlier?

@BKreisel, could you check it, please? :)

**Comment by BKreisel** [comment]
LGTM as well. I've confirmed the issue outlined in #841 and that this patch resolves the issue.

**Comment by davidxia** [comment]
@kuba I've reordered the imports. Thanks.

**Comment by bmw** [comment]
Thanks for the patch @davidxia.

lgtm

**Pull request merged**